packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -36,6 +37,24 @@ abstract class PatrolCommand extends Command<int> {
     );
   }
 
+  void usesBuildModeOption() {
+    _usesBuildOption = true;
+    argParser
+      ..addFlag(
+        'debug',
+        help: 'Build a debug version of your app',
+        defaultsTo: true,
+      )
+      ..addFlag(
+        'profile',
+        help: 'Build a version of your app for performance profiling.',
+      )
+      ..addFlag(
+        'release',
+        help: 'Build a release version of your app (default mode)',
```

Comment:
But `debug` has `defaultsTo: true` o_O

Reply:
oopsie, copied from `flutter build ios` and forgot to change